Riot/Modules/Settings/Security/ManageSession/ManageSessionViewController.m

```diff
@@ -0,0 +1,789 @@
+/*
+ Copyright 2020 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "ManageSessionViewController.h"
+
+#import <MatrixKit/MatrixKit.h>
+
+#import <OLMKit/OLMKit.h>
+
+#import "AppDelegate.h"
+#import "AvatarGenerator.h"
+
+#import "ThemeService.h"
+
+#import "Riot-Swift.h"
+
+
+enum
+{
+    SECTION_SESSION_INFO,
+    SECTION_ACTION,
+    SECTION_COUNT
+};
+
+enum {
+    SESSION_INFO_SESSION_NAME,
+    SESSION_INFO_TRUST,
+    SESSION_INFO_COUNT
+};
+
+enum {
+    ACTION_REMOVE_SESSION,
+    ACTION_COUNT
+};
+
+
+@interface ManageSessionViewController () <
+MXKDataSourceDelegate,
+MXKDeviceViewDelegate,
```

Comment:
In the current implementation `DeviceView` is not used. Did you keep this dependence for the coming changes (see "TODO with bottom sheet"